src/msg.rs

```diff
@@ -74,7 +74,7 @@ impl Validate for InstantiateMsg {
             (Some(_), Some(_)) => (),
             (None, None) => (),
         }
-        if self.price_precision.lt(&Uint128::new(0)) || self.price_precision.gt(&Uint128::new(18)) {
+        if self.price_precision.gt(&Uint128::new(18)) {
```

Comment:
mind explaining why the 0 check was removed?

Reply:
uint is unsigned, so has to be > 0 otherwise i should panic